src/scripts/interfaces/templates.ts

```diff
@@ -54,7 +54,7 @@ export interface Templates {
 
   choiceGroup(options: TemplateOptions, group: GroupFull): HTMLDivElement;
 
-  choice(options: TemplateOptions, choice: ChoiceFull, selectText: string, groupText?: string): HTMLDivElement;
+  choice(options: TemplateOptions, choice: ChoiceFull, selectText: string, deselectText: string, groupText?: string): HTMLDivElement;
```

Comment:
Instead of an argument, please add it as part of the `TemplateOptions` to avoid a breaking change to the function signature